### ISSUE TITLE
ClassName prop on container element

### ISSUE THREAD

**Issue by kaliwa** [dennismorello/react-awesome-reveal#41]
Previously that helped a lot, but now I understand that what I need is the effect on the container element, not on the child element.
I've tried version 2 to find out, but this version 3 updates is very good, and it's a shame if I have to go back to version 2. Sorry for bothering you

**Comment by dennismorello** [comment]
The container element only implements the animations, can you please share a snippet with a description of what you need?

**Comment by kaliwa** [comment]
```
<Fade className='row'  triggerOnce cascade>
            {properties.map((el, i) => (
			<Col md={3} key={i} xs={6} className='mb-3'>
				<Someproperty {...el} />
			 </Col>
             ))}
</Fade>
```
I'm using version 2, and react-bootstrap. And everything works as it should.

**Comment by dennismorello** [comment]
Ok, to let things be as broader as possible, the final solution I propose is:

- `className` and `style` are added to the container element
- `childClassName` and `childStyle` are added to the child element

A minor version will be released soon.

**Comment by kaliwa** [comment]
Okay, that's much better. Thank you

**Comment by dennismorello** [comment]
Implemented in [v3.1.1](https://www.npmjs.com/package/react-awesome-reveal/v/3.1.1).
